main.rb

```diff
@@ -0,0 +1,40 @@
+require './src/app'
+
+def present_options
+  puts "
+  Choose an option:
+
+  1. List Items
+  2. Add a new Book
+  3. Add a new Music album
+  4. Add a new Movie
+  5. Add a new Game
+  6. Exit"
+end
```

Comment:
There's an error when saving data on exit.

![image](https://user-images.githubusercontent.com/38088818/202695033-715bb747-0fbe-47d7-af06-66094224c490.png)

- That's because the `Json` directory with the respective `/*.json` file does not exist in your project. You might want to create a method that validates if the particular file you want to save exists first before saving.

Something like:
```
  def validate_file(file_name)
    if File.exist?("./json/#{file_name}.json")
      #Save data method
    else
      Dir.mkdir('json') unless Dir.exist?('json')

      File.open(path, 'w') do |file|
        file.puts JSON.generate([])
      end
    end
  end
```

Reply:
Thanks for the review I am going to address all these changes